pkg/model/components/networking.go

```diff
@@ -69,5 +69,18 @@ func (b *NetworkingOptionsBuilder) BuildOptions(o interface{}) error {
 		}
 	}
 
+	if networking.Romana != nil {
+		daemonIP, err := WellKnownServiceIP(clusterSpec, 99)
```

Comment:
You may want to set these only if they aren't already set, to allow the user to override them if they want to.  That said I suspect we don't want to allow the user to override them?

Reply:
"Our" needs for the address is mainly that it's static and reserved, because in this "layer" we can't depend on kube-dns pods being reachable.
I don't think it'd be an issue if users changed them, because they'd get substituted in all the necessary places, but it's not really a valuable thing to do.

Reply:
@justinsb thoughts?